lib/twemoji.rb

```diff
@@ -1,3 +1,4 @@
+# encoding: utf-8
```

Comment:
You are correct, [default is UTF-8 after 2.0](http://ruby-doc.org//core-2.2.0/Encoding.html#class-Encoding-label-Script+encoding)